src/server/bitops_family.cc

```diff
@@ -38,7 +44,7 @@ void BitOp(CmdArgList args, ConnectionContext* cntx);
 void GetBit(CmdArgList args, ConnectionContext* cntx);
 void SetBit(CmdArgList args, ConnectionContext* cntx);
 
-OpResult<std::string> ReadValue(const OpArgs& op_args, std::string_view key);
+OpResult<std::string> ReadValue(EngineShard* shard, const DbContext& context, std::string_view key);
```

Comment:
output (mutable) arguments should be last one in the function.

Reply:
will change